documentation/src/main/asciidoc/topics/con_protobuf_encoding.adoc

```diff
@@ -0,0 +1,52 @@
+[id='protobuf-encoding_{context}']
+= Protobuf cache encoding
+Protocol Buffers (Protobuf) is a lightweight binary media type for structured data.
+As a cache encoding, Protobuf gives you excellent performance as well as interoperability between client applications in different programming languages for both Hot Rod and REST endpoints.
+
+The following example shows a Protobuf message that describes a `Person` object:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+include::protobuf/personalshopper.proto[]
+----
+
+[NOTE]
+====
+{brandname} uses a ProtoStream library to encode caches with the `application/x-protostream` media type.
+This encoding is similar to `application/x-protobuf` but the two media types have some differences and are not interchangeable.
```

Comment:
Last time I looked, there was not an "official" Protobuf media type, I've seen a handful of media types being used for protobuf, so better not to make promises here :smile: 
I will check again, but nothing prevents adding an alias to an eventual standard for protobuf in the future.

```suggestion
{brandname} uses the media type `application/x-protostream` for Protobuf encoded caches. 
```

Reply:
I've added this to the new PR.